backend/apid/routers/entities.go

```diff
@@ -3,22 +3,27 @@ package routers
 import (
 	"github.com/gorilla/mux"
 	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/backend/apid/actions"
 	"github.com/sensu/sensu-go/backend/apid/handlers"
 	"github.com/sensu/sensu-go/backend/store"
 )
 
 // EntitiesRouter handles requests for /entities
 type EntitiesRouter struct {
-	handlers handlers.Handlers
+	handlers   handlers.Handlers
+	store      store.Store
+	eventStore store.EventStore
```

Comment:
Since `store.Store` adheres to the `store.EventStore` interface, could we just keep `store.Store` in the router and pass it as the EntityStore & EventStore below, instead of having to deal with both stores in the router?

Reply:
No, it needs to be its own thing due to pluggable event stores.

Reply:
I was also curious about the 2 stores